src/__tests__/components/pages/state/screenshots.js

```diff
@@ -1,12 +1,10 @@
 import React from 'react'
 import renderer from 'react-test-renderer'
-import Enzyme, { shallow } from 'enzyme'
-import Adapter from 'enzyme-adapter-react-16'
```

Comment:
Because we now have a setup file, we don't need these in each test file where we want to use enzyme.

Reply:
Oh, nice!